regex/README.md

```diff
@@ -0,0 +1,9 @@
+name: Regex
+
+description: Match, extract and replace text-based data based on patterns.
+
+icon: 
```

Comment:
@kapnobatai137 Wasn't sure what to use here. If you have time tomorrow please could you walk me through the process of how we store images?